Library/Formula/joshua-v.rb

```diff
@@ -0,0 +1,48 @@
+class JoshuaV < Formula
+  desc "Joshua is an open-source statistical machine translation decoder for phrase-based, hierarchical, and syntax-based machine translation, written in Java."
```

Comment:
Please use a shorter description. “`joshua-v: …desc…`” should fit under 80 characters.